setup.cfg

```diff
@@ -103,7 +103,7 @@ tests =
     # https://github.com/pytest-dev/pytest/issues/12263
     pytest>=6,<8.2
     towncrier>=23
-    types-pkg_resources>=0.1.2
+    types-setuptools
```

Comment:
> mypy passes without it

Not sure if that means removing the dep removes (some) type checking?

```suggestion
```

?

Reply:
Not sure, I'll just merge this as-is